examples/window-app/common/window-app.zap

```diff
@@ -840,7 +840,7 @@
               "reportableChange": 0
             },
             {
-              "name": "PartsList",
+              "name": "parts list",
```

Comment:
Why ?

Reply:
This already bothering to not have this properly set in the doc and in chip tool 
Specification used CamelCase therefore everything should be camel case and our effort should be to correct this files to be CamelCased